tutorials/nservicebus-101/index.md

```diff
@@ -4,32 +4,19 @@ suppressRelated: true
 reviewed: 2017-01-26
 ---
 
-In this course, you will learn how to get working quickly with NServiceBus.
+partial: nsb101-intro-paragraph
 
+ * In [Lesson 1: Getting started](lesson-1/), which should take no longer than 10-15 minutes, you will learn how to set up your development environment and create your very first messaging endpoint.
+ * In [Lesson 2: Sending a command](lesson-2/), which should take no longer than 15-20 minutes, you will learn how to define messages and message handlers. In the exercise, you'll send and process a `PlaceOrder` command within the same endpoint.
```

Comment:
The exercise descriptions don't really add value if you aren't familiar with the material or terminology. The intro paragraph lets them know they'll be actually building something.